src/lib/screens/explore.dart

```diff
@@ -1,86 +1,102 @@
-
+import 'package:ESOF/model/conference.dart';
+import 'package:ESOF/screens/post.dart';
 import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:flutter/material.dart';
 
-class ExploreScreen extends StatelessWidget{
+import '../style.dart';
 
+class ExploreScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         title: Text("Search Conferences"),
         actions: <Widget>[
-          IconButton(icon: Icon(Icons.search), onPressed: () => {
-            showSearch(context: context, delegate: ConferenceSearch())
-          })
+          IconButton(
+              icon: Icon(Icons.search),
+              onPressed: () =>
+                  {showSearch(context: context, delegate: ConferenceSearch())})
         ],
+        backgroundColor: accentOrange,
       ),
-      drawer: Drawer(),
     );
   }
 }
 
 class ConferenceSearch extends SearchDelegate<String> {
-  final _allConferences = ["HELLO","MEGA CONFERENCIA", "WE LOVE YOU_ CONF"];
-  final _lastConferences = ["HELLO"];
+  // If the conference has image return the url, otherwise returns a default.
+  // Mudar para ficar mais modular!
+  hasImage(conference) {
+    return conference['img'] == null
+        ? 'http://www.theides.org/img/about.jpg'
+        : conference['img'];
+  }
```

Comment:
This function already exists in https://github.com/FEUP-ESOF-2020-21/open-cx-t4g2-codemasters/blob/explore/src/lib/widgets/common/feed_common.dart

Reply:
Sim eu fui buscar lá. Como uso a função sem a definir duas vezes?
Acho que esse problema vai ficar resolvido quando a Search Bar passar para o Home

Reply:
Basta retirares a definição da função, e fazeres include do ficheiro onde a definição da função está

Reply:
Done!